webhook/logging/logging.go

```diff
@@ -31,11 +33,15 @@ func WebhookLogger(errorType string, errorMessage error) error {
 
 	file, err := os.OpenFile(logFileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 	if err != nil {
+		fmt.Print("fmt: \n", err)
```

Comment:
Can you also be more explicit about the file open error description, like for example  "Failed to open log file"

Reply:
Sure!